ddtrace/tracer.py

```diff
@@ -367,7 +367,7 @@ def configure(
             # No need to do anything for the LogWriter.
             pass
         if isinstance(self.writer, AgentWriter):
-            self.writer.dogstatsd = get_dogstatsd_client(self._dogstatsd_url)
+            self.writer.dogstatsd = get_dogstatsd_client(self._dogstatsd_url)  # type: ignore[has-type]
```

Comment:
what changed that needed this?

Reply:
will investigate

Reply:
bump here just to get a resolution 🙂

Reply:
My `mypy` is not happy without it 🙁

Reply:
<img width="1071" alt="Screenshot 2021-07-08 at 11 47 00" src="https://user-images.githubusercontent.com/20231758/124909577-83b1c280-dfe2-11eb-8ba0-7486853b202d.png">